js/views/userPageVw.js

```diff
@@ -1413,6 +1437,92 @@ module.exports = baseVw.extend({
 
   },
 
+  addModeratorClick: function(e){
+    var $targ = $(e.target).closest('.js-addmoderator');
+
+    $targ.addClass('loading');
+
+    var user = this.model.get('user');
+    var self = this;
+
+    var modList = {};
+    modList.moderators = user.moderators;
+    modList.moderators[modList.moderators.length] = this.userProfile.get('profile').guid;
+    user.moderators = modList.moderators;
+
+    saveToAPI('', user, user.serverUrl + "settings",
+      function(){
+        // confirmed
+        $targ.removeClass('loading');
+      },
+      function(){
+        // failed
+      }, modList,'',
+      function(){
```

Comment:
Leave the invalid function as null or '' too, otherwise you're overriding the defaults with a function that doesn't do anything.